docusaurus/docs-internals/editor-controls/multivalue-controls/sidebarTabsButton.md

```diff
@@ -0,0 +1,179 @@
+| :-------------- | :--------------------------------------- | :--------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`            | `string`                                 |     -      | The identifier of the key where the sidebarTabsButton will save your data                                                                                                                                                        |
+| `type`          | `string`                                 |     -      | Type should be `"sidebarTabsButton"` to use this control                                                                                                                                                                         |
+| `devices?`      | `"all"` \| `"desktop"` \| `"responsive"` |  `"all"`   | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices |
+| `disabled?`     | `boolean`                                |  `false`   | Configure the condition under which the control is disabled or enabled                                                                                                                                                           |
+| `config?.tabId` | `string`                                 |     -      | The identifier of the tab where the control will be placed                                                                                                                                                                       |
+| `config?.icon`  | `string`                                 |     -      | The icon to be displayed with the control                                                                                                                                                                                        |
+| `config?.align` | `"left"` \| `"center"` \| `"right"`      | `"center"` | The alignment of the control                                                                                                                                                                                                     |
+| `config?.text`  | `string`                                 |     -      | The text to be displayed with the control                                                                                                                                                                                        |
+
+### Return value
```

Comment:
as i see this control does not return anything

![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/911b97f4-5bb6-482e-a7f3-f0070816264a)